units/serial-getty@.service.in

```diff
@@ -11,7 +11,8 @@
 Description=Serial Getty on %I
 Documentation=man:agetty(8) man:systemd-getty-generator(8)
 Documentation=https://0pointer.de/blog/projects/serial-console.html
-BindsTo=dev-%i.device
+PartOf=dev-%i.device
+ConditionPathExists=/dev/%i
```

Comment:
The device may not exist yet when the condition is examined, but may appear later. That can easily happen e.g. when the device is in USB bus.